Docs/Configuration.tex

```diff
@@ -3186,98 +3208,101 @@ \subsection{Boot Properties}\label{miscbootprops}
   \texttt{TakeoffDelay}\\
   \textbf{Type}: \texttt{plist\ integer}, 32 bit\\
   \textbf{Failsafe}: \texttt{0}\\
-  \textbf{Description}: Delay in microseconds performed before handling
-  picker startup and \texttt{action hotkeys}.
+  \textbf{Description}: Delay in microseconds executed before handling
+  boot picker startup and \texttt{action hotkeys}.
 
   Introducing a delay may give extra time to hold the right \texttt{action hotkey}
-  sequence to e.g. boot to recovery mode. On some platforms setting this option to
-  at least \texttt{5000-10000} microseconds may be necessary to access
-  \texttt{action hotkeys} at all due to the nature of the keyboard driver.
+  sequence to, for instance, boot into recovery mode. On some platforms, setting this
+  option to a minimum of \texttt{5000-10000} microseconds may be required to access
+  \texttt{action hotkeys} due to the nature of the keyboard driver.
 
 \item
   \texttt{Timeout}\\
   \textbf{Type}: \texttt{plist\ integer}, 32 bit\\
   \textbf{Failsafe}: \texttt{0}\\
   \textbf{Description}: Timeout in seconds in boot picker before
-  automatic booting of the default boot entry. Use 0 to disable timer.
+  automatic booting of the default boot entry. Set to \texttt{0} to disable.
 
 \item
   \texttt{PickerMode}\\
   \textbf{Type}: \texttt{plist\ string}\\
   \textbf{Failsafe}: \texttt{Builtin}\\
   \textbf{Description}: Choose boot picker used for boot management.
 
-  Picker describes underlying boot management with an optional user interface
-  responsible for handling boot options. The following values are supported:
+  PickerMode describes the underlying boot management with an optional user
+  interface responsible for handling boot options.
+
+  The following values are supported:
 
   \begin{itemize}
   \tightlist
   \item \texttt{Builtin} --- boot management is handled by OpenCore, a simple
-  text only user interface is used.
+  text-only user interface is used.
   \item \texttt{External} --- an external boot management protocol is used
-  if available. Otherwise \texttt{Builtin} mode is used.
+  if available. Otherwise, the \texttt{Builtin} mode is used.
   \item \texttt{Apple} --- Apple boot management is used if available.
-  Otherwise \texttt{Builtin} mode is used.
+  Otherwise, the \texttt{Builtin} mode is used.
   \end{itemize}
 
-  Upon success \texttt{External} mode will entirely disable all boot management
-  in OpenCore except policy enforcement. In \texttt{Apple} mode it may additionally
-  bypass policy enforcement. See \hyperref[ueficanopy]{OpenCanopy} plugin
-  for an example of a custom user interface.
+  Upon success, the \texttt{External} mode will entirely disable all boot management
```

Comment:
will → may. OpenCanopy was changed to actually reuse most of that.

Reply:
Changed. Use of 'may' might be reserved later.